app/pages/addorg/contextCreator.test.js

```diff
@@ -0,0 +1,21 @@
+import manifest from './manifest.json';
+import { getContext } from './contextCreator';
+
+describe('addorg contextCreator', () => {
+  describe('getContext', () => {
+    it('should return the contents of the manifest', () => {
+      const context = getContext({});
+      expect(context.title).toEqual(manifest.title);
+      expect(context.description).toEqual(manifest.description);
+      expect(context.questions.length).toEqual(manifest.questions.length);
+      expect(context.questions[0].id).toEqual(manifest.questions[0].id);
+      expect(context.questions[0].text).toEqual(manifest.questions[0].text);
```

Comment:
should that be `mainAdvice` rather than `text`?

Reply:
yes!